server/user.go

```diff
@@ -33,8 +33,14 @@ type Connection struct {
 	Oauth1AccessSecret string        `json:",omitempty"`
 	OAuth2Token        *oauth2.Token `json:",omitempty"`
 	Settings           *ConnectionSettings
-	DefaultProjectKey  string   `json:"default_project_key,omitempty"`
-	MattermostUserID   types.ID `json:"mattermost_user_id"`
+	SavedFieldValues   *SavedFieldValues `json:"saved_field_values,omitempty"`
```

Comment:
We are losing data here by changing its shape, but the "old" value of `DefaultProjectKey` is not too important. Just something to think about when changing persistent data structures